lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -283,3 +283,19 @@ def __init__(
         **kwargs,
     ):
         super().__init__(child, f=standardizer.standardize, args=path, kwargs=kwargs)
+
+class IgnoreErrors(standardizer)
```

Comment:
Technically this is not a standardizer, so no need to subclass it. Also colon at the end

Reply:
Ack